logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionMapper.kt

```diff
@@ -30,7 +33,7 @@ internal class ConnectionMapperImpl(
     override fun fromApiToDao(state: ConnectionDTO): ConnectionEntity = ConnectionEntity(
         conversationId = state.conversationId,
         from = state.from,
-        lastUpdate = state.lastUpdate,
+        lastUpdateInstant = state.lastUpdate.takeIf { it.isNotBlank() }?.toInstant() ?: Instant.UNIX_FIRST_DATE,
```

Comment:
maybe as an extension fun to all instants?

Reply:
I did it for the `ConnectionMapper`, but this is not currently used elsewhere and I'd prefer to remove the passing of date strings around and remove the need for this parsing completely instead of making this function public